gradle.properties

```diff
@@ -3,19 +3,24 @@ org.gradle.jvmargs = -Xmx1G
 
 # Fabric Properties
 # Check these on https://modmuss50.me/fabric.html
-minecraft_version = 1.16.3
-yarn_mappings = 1.16.3+build.11
+minecraft_version = 1.16.5
+yarn_mappings = 1.16.5+build.10
 loader_version = 0.9.3+build.207
 
 #Fabric api
-fabric_version = 0.21.0+build.407-1.16
-loom_version = 0.5-SNAPSHOT
+fabric_version = 0.41.3+1.16
+#Last loom version that does not require java 16
+loom_version = 0.7-SNAPSHOT
 
 # Mod Properties
 mod_version = 1.0.0
 maven_group = net.logandark
 archives_base_name = barebones
+mod_id = barebones
```

Comment:
You should probably just use archives_base_name

Reply:
Done in 69d1be1